src/intf/interface.cpp

```diff
@@ -8,26 +8,26 @@ INT32 IntInfoFree(InterfaceInfo* pInfo)
 		for (INT32 i = 0; i < INT_INFO_STRINGS; i++) {
 			if (pInfo->ppszInterfaceSettings[i]) {
 				free(pInfo->ppszInterfaceSettings[i]);
-				pInfo->ppszInterfaceSettings[i] = NULL;
+				pInfo->ppszInterfaceSettings[i] = nullptr;
 			}
```

Comment:
we also can't support nullptr due to the same reason (backwards compiler compatibility).  It's very important that fb stays compatible with ancient hw :)

Reply:
I'm 99% positive it does improve the experience on Windows OS

Reply:
Is it possible that it's fixed that vs2019 extra line issue #132?

Reply:
you could make a global variable for the make file "ancient_cpp" and define it for platforms that need it else it uses the more optimized route for later platforms nothing wrong with supporting both.

Reply:
![Capture](https://user-images.githubusercontent.com/50218879/71544393-39786b80-297f-11ea-95c4-5de951f0ebbb.PNG)

Reply:
we're not going to be changing the c++standard or modernizing anything, sorry about that.

In what way does changing NULL to nullptr improve the experience in windows?  I think that blank line in vs is coming from a bug in scrnsize in burner/win32/scrn.cpp

Reply:
@dinkc64 

I can revert the nullptr changes, if you wish.